content/How_To/environment/Skybox.md

```diff
@@ -44,11 +65,13 @@ skyboxMaterial.diffuseColor = new BABYLON.Color3(0, 0, 0);
 skyboxMaterial.specularColor = new BABYLON.Color3(0, 0, 0);
 skybox.material = skyboxMaterial;
 ```
+<br/>

```

Comment:
Here's how I like it to look (this is with the HTML \<br\>):

![image](https://user-images.githubusercontent.com/279819/119244913-ec73d700-bb29-11eb-85b9-d853730cba82.png)

Here's how it looks if I take out the \<br\>:

![image](https://user-images.githubusercontent.com/279819/119244930-0ad9d280-bb2a-11eb-9e73-f42a90356f74.png)

The spacing there is weird, and markdown blank lines don't fix it. I'd love to get tips from @RaananW or others on a better technique. As far as I can tell the core problem is that in the custom CSS, \<p\> blocks get spacing below but not above, but this isn't consistent with other block-type elements such as \<pre\>, which means that the end of a non-\<p\> and the start of the next \<p\> tend to smush together. Personally, I'd change the CSS so there's collapsible margin before and after \<p\> (which is probably the default) but then a lot of pages would probably require minor tweaks.

(This comment applies to all my other insertions of \<br\> or \<p\>, so I'll leave those unresolved for now.)

Reply:
Adding @RaananW again for guidance here.

Reply:
This feels like a missing css definition. Which has something to do with the <pre>  tag (code container). Are the other <br> elements the same?

Reply:
I added a margin to all `<pre>` element, the breaks are not required anymore.

Reply:
(Sorry for the high latency!)

I've removed the \<br/\> tags after preformatted text, but text following tables and \<Playground\> tags still seem to require explicit HTML for reasonable spacing (the example images are what happens _without_ \<br\> or \<p\>):

![image](https://user-images.githubusercontent.com/279819/121786789-06c43200-cb77-11eb-805f-ba29933345e3.png)

![image](https://user-images.githubusercontent.com/279819/121786865-7afed580-cb77-11eb-902c-e547908943eb.png)

Is there an alternative? I could also just stop being fussy, but I think it can be a little confusing when text is mushed into an object like that, it makes it look more like a caption.

Reply:
I will make sure to add those styles after this is merged. Let's not make this the issue to prevent the changes from merging.